plugins/inputs/kube_state/node.go

```diff
@@ -0,0 +1,56 @@
+package kube_state
+
+import (
+	"context"
+
+	"github.com/ericchiang/k8s/apis/core/v1"
+
+	"github.com/influxdata/telegraf"
+)
+
+func collectNodes(ctx context.Context, acc telegraf.Accumulator, ks *KubernetesState) {
+	list, err := ks.client.getNodes(ctx)
+	if err != nil {
+		acc.AddError(err)
+		return
+	}
+	for _, n := range list.Items {
+		if err = ks.gatherNode(*n, acc); err != nil {
+			acc.AddError(err)
+			return
+		}
+	}
+}
+
+func (ks *KubernetesState) gatherNode(n v1.Node, acc telegraf.Accumulator) error {
+	fields := map[string]interface{}{}
+	tags := map[string]string{
+		"node_name": *n.Metadata.Name,
+	}
+
+	for resourceName, val := range n.Status.Capacity {
+		switch resourceName {
+		case "cpu":
+			fields["status_capacity_cpu_cores"] = atoi(*val.String_)
```

Comment:
`.String()` returns `string:"16" ` (trailing space included) whereas just accessing `.String_` returns `16`. I saw docs refer to the `Int64()` function, never saw it in code, but figured it would have returned `int64: 16` or something as equally useless

Reply:
Ah, how'd I miss that, it looks like that should work